src/Components/LogInForm.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import PrimaryButton from './PrimaryButton';
+import CloseIcon from './CloseIcon';
+import { Link } from 'react-router-dom';
+import HorizontalLine from './HorizontalLine';
+import { useNavigate } from 'react-router-dom';
+
+const LogInForm = ({ setIsLoggedIn }) => {
+  const navigate = useNavigate(); 
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+    alert(`Logging in...`);
+    setIsLoggedIn(true);
+    navigate("/home");
+  };
+
+  return (
+    <div style={styles.container}>
+      <form onSubmit={handleLogin} style={styles.form}>
+        <Link to="/">
+          <CloseIcon />
+        </Link>
+        <h2 style={styles.title}>Welcome Back</h2>
+        {/* <div style={styles.lineSeparator}></div> */}
+        <HorizontalLine width="400px"/>
+        <div style={styles.fieldContainer}>
+          <label htmlFor="email" style={styles.label}>Email</label>
+          <input
+            style={styles.input}
+            type="text"
+            placeholder="Email"
+            value={username}
+            onChange={(e) => setUsername(e.target.value)}
+          />
+        </div>
+
+        <div style={styles.fieldContainer}>
+          <label htmlFor="password" style={styles.label}>Password</label>
+          <input
+            style={styles.input}
+            type="password"
+            placeholder="Password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+          />
+        </div>
+
+        <div style={styles.fieldContainer}>
+          <p><u>Forgot password?</u></p>
+        </div>
+
+        {/* Primary Button */}
+        <PrimaryButton type="submit">Log In</PrimaryButton>
+      </form>
+    </div>
+  );
+};
+
+// Styling
+const styles = {
```

Comment:
Another reminder about removing inline styling and using styled components instead, where you can write the correct css attributes.